src/Sql/SqlMysql.php

```diff
@@ -17,15 +21,19 @@ class SqlMysql extends SqlBase
      */
     public static function make(array $dbSpec, array $options)
     {
-        // First get a MySQL instance
-        $instance = new self($dbSpec, $options);
+        // If the mysql version reports that it is MariaDB, use MariaDB as client.
+        $process = Drush::shell('mysql --version');
+        $process->setSimulated(false);
+        $process->run();
+        if ((!$process->isSuccessful() || strpos($process->getOutput(), 'MariaDB') !== false) && self::programExists('mariadb')) {
```

Comment:
I can't figure out what the double parens do here. Did you mean to surround so that the `||` condition is more clearly evaluated? Also, lets keep using str_contains() and not strpos()

Reply:
Also needs code style fix

`PSR12.Classes.OpeningBraceSpace.Found at /home/wodby/drush/src/Sql/SqlMariaDB.php (8:1)`

Reply:
`||` and `&&` ordering is not always clear to read and changes in every language, so my nature makes them explicit so when im changing languages there is no question if the order is correct